lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -106,6 +111,7 @@ public void Submit(IJobSubmission jobSubmission)
                 javaParams.TcpPortRangeCount.ToString(),
                 javaParams.TcpPortRangeTryCount.ToString()
                 );
+            _jobStatus = State.SUBMITTED;
```

Comment:
You are right. We don't have simulation for getting Job status in Local runtime as provided by Yarn Rest API. This is something we need to put in TODO list. I have create REEF-889 for it.